cmds/core/ip/neigh_linux_test.go

```diff
@@ -129,9 +129,16 @@ func TestParseNeighAddDelReplaceParam(t *testing.T) {
 }
 
 func TestParseNeighShowFlush(t *testing.T) {
```

Comment:
Would it make sense to add tests for IPv6 as well, or is this redundant?

Reply:
Added :+1: